tests/run.sh

```diff
@@ -86,13 +86,6 @@ echo "Detected Mender branch: $MENDER_BRANCH"
 echo "Detected mender-artifact branch: $MENDER_ARTIFACT_BRANCH"
 echo "Detected mender-cli branch: $MENDER_CLI_BRANCH"
 
-function inject_pre_generated_ssh_keys() {
```

Comment:
Nice, but see comment from the last commit. I think you should remove them both here